pkg/unshare/unshare_linux.go

```diff
@@ -427,6 +427,15 @@ func IsRootless() bool {
 				isRootless = true
 			}
 		}
+		if !isRootless {
```

Comment:
Exactly this check already exists a few lines above — don’t we only need this once?